docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
+port. Almost any USB Wi-Fi dongle which supports Linux will work, including the
+one that is supported by the official LEGO software. Once you have plugged in a
+dongle, you can use Brickman's "**_Wireless and Networks_ > _Wi-Fi_**" menu to
+connect to a network.
+
+### With a USB Ethernet adapter
```

Comment:
I agree that it should be below most of the other options, however I think it should still be above the tethering ones, as they're not helpful for most cases. Inevitably we're going to tell people to upgrade something, so I'd rather have them using a method which supplies internet access than have to instruct them to change their set-up and then try again. What do you think about putting it after the USB and Bluetooth internet ones but before the tethering options?